WordPress/Classes/ViewRelated/Me/My Profile/Gravatar/GravatarQuickEditorPresenter.swift

```diff
@@ -16,9 +17,24 @@ struct GravatarQuickEditorPresenter {
         }
         self.email = email
         self.authToken = account.authToken
+        self.emailVerificationStatus = account.verificationStatus
```

Comment:
Hey, I'm not sure how accurate this field is. The usage of it was removed from the app, and I was planning to remove the field as well in favor of relying on the server to tell you which operations are not permitted without hardcoding it on the client.

Reply:
> What happens if the client doesn't hardcode this check?

Sounds good to me but is there a way to currently achieve that on the JP side? 

We want to improve our endpoints to know the reason as to why 401 unauthorized is thrown. That way we'll be able to show a better error. Unfortunately we didn't have chance to come up with a better solution on the SDK side yet(holidays).

There has been a very recent backend change on the Gravatar side. Before that, the Quick Editor worked successfully for unverified emails, but it turns out this isn't supposed to be happening so it's fixed urgently. So, a "session expired" error is displayed in the Gravatar Quick Editor for unverified emails which can be misleading and confusing. This is just an attempt to show a better message until we sort things out the SDK side. 

Although I am not sure when this change would be live even if we merged this...